File: LovePlay-UIKit/LovePlay/Class/Zone/Model/DiscussListModel.m
//
//  DiscussListModel.m
//  LovePlay
//
//  Created by Yuns on 2017/2/2.
//  Copyright © 2017年 Yuns. All rights reserved.
//

#import "DiscussListModel.h"

@implementation DiscussListModel

+ (NSDictionary *)modelContainerPropertyGenericClass
{
    return @{@"forum_threadlist" : [ForumThread class]};
}


@end

@implementation ForumThread

+ (NSDictionary *)modelCustomPropertyMapper
{
    return @{@"typeId" : @"typeid"};
}

@end

File: LovePlay-Swift3/LovePlay/Class/Zone/Model/DiscussListModel.swift
//
//  DiscussListModel.swift
//  LovePlay
//
//  Created by weiying on 2017/6/6.
//  Copyright © 2017年 yuns. All rights reserved.
//

import UIKit
import HandyJSON

class DiscussListModel: HandyJSON {
    var tid : String?
    var typeid : String?
    var author : String?
    var authorid : String?
    var subject : String?
    var dateline : String?
    var lastpost : String?
    var lastposter : String?
    var replies : String?
    var displayorder : Int?

    required init() {
        
    }
}
